packages/web/tailwind.config.js

```diff
@@ -5,19 +5,28 @@ module.exports = {
     colors: {
       white: '#FFFFFF',
       gray: {
-        50: '#F5F5FB',
-        100: '#EBEBF2',
-        200: '#D9D9D9',
-        300: '#AAAAAA',
+        100: '#FAFAFF',
+        200: '#EBEBF2',
+        300: '#D8D9E0',
+        400: '#B4B5BD',
         500: '#777777',
-        600: '#555555',
-        700: '#333333',
+        600: '#555968',
+        700: '#3A3F57',
+        800: '#252946',
+        900: '#161A3A',
       },
+      lightgreen: '#2EEFAA',
       green: '#00D287',
-      darkgreen: '#00D287',
+      darkgreen: '#00B372',
+      lightblue: '#75BDFF',
       blue: '#1C92FF',
+      darkblue: '#006DD1',
+      lightviolet: '#C1B7FF',
       violet: '#9F8FFF',
-      red: '#FF4D4D',
+      darkviolet: '#7A68E3',
+      lightred: '#FFA8A8',
+      red: '#FF8787',
+      darkred: '#C93939',
```

Comment:
```suggestion
      darkViolet: '#7A68E3',
      lightRed: '#FFA8A8',
      red: '#FF8787',
      darkRed: '#C93939',
```
아마 각각 `violet-dark`, `red-light` 등으로 바뀌긴할테지만 혹시 변수명을 지어야 할 때 `camelCase` 로 통일하는게 어떨까여?
현재 컴포넌트 파일명은 맨 앞글자 대문자인거 같고, 컴포넌트명은 `CamelCase` 로 맞춰져있는거같은데 혹싀나 ㅎㅎ

Reply:
이 부분은 #57 커밋이 머지되면서 변경되었습니다!

https://github.com/80000Coding/80000Coding-Web-Client/blob/10014007b36eb1b6233f52b713fa4b2ccbc32a07/packages/web/tailwind.config.js#L18-L37